src/data_watcher.hpp

```diff
@@ -121,6 +121,13 @@ class DataWatcher
      */
     uint32_t _eventMasksToWatch;
 
+    /**
+     * @brief The group of events for which the data need to be watched on the
+     * parent if the configured path is not exisitng in the filesystem.
```

Comment:
You could add details like why one more member with fixed event rather than using the given events to this class.